os/arch/arm/Kconfig

```diff
@@ -45,6 +46,20 @@ config ARCH_CHIP_STM32
 	select ARCH_HAVE_HEAPCHECK if DEBUG
 	---help---
 		STMicro STM32 architectures (ARM Cortex-M3/4).
+		
+config ARCH_CHIP_IMXRT
+	bool "NXP/Freescale iMX.RT"
+	select ARCH_CORTEXM7
+	select ARCH_HAVE_MPU
+	select ARCH_HAVE_FETCHADD
```

Comment:
Is this config required ?

Reply:
In build/configs/imxrt1050-evk/nxp_demo/defconfig
CONFIG_ARCH_CHIP_IMXRT is defined normally.

Reply:
I dont think this configuration is required as of now

Reply:
do you mean that ARCH_HAVE_FETCHADD define is not required?

If so, I will remove it